src/current/v25.2/create-table-as.md

```diff
@@ -287,6 +289,34 @@ You can define the [column families]({% link {{ page.version.version }}/column-f
 (1 row)
 ~~~
 
+### Populate `CREATE TABLE AS` with historical data using `AS OF SYSTEM TIME`
+
+{% include_cached new-in.html version="v25.2" %} CockroachDB supports creating a table using historical data using the [`AS OF SYSTEM TIME`]({% link {{ page.version.version }}/as-of-system-time.md %}) clause. You can use this to create a new table based on the state of an existing table as of a specific [timestamp]({% link {{ page.version.version }}/timestamp.md %}) in the past. This is useful for:
+
+- Generating static datasets for reporting or analytical workloads without increasing contention on production tables or otherwise impacting performance.
+- Analyzing data changes over time.
+- Preserving historical data for regulatory or investigative purposes.
```

Comment:
We should specifically call out one more use case here, specifically undoing an accidental table deletion with an example.

```
-- Step 1: Create and populate the original table
CREATE TABLE customers (
    id INT PRIMARY KEY,
    name STRING
);

INSERT INTO customers (id, name) VALUES (1, 'Alice'), (2, 'Bob');

-- Step 2: Get the timestamp from before deletion
SELECT now() AS restore_ts;

-- Step 3: Wait 5 seconds to simulate time passage (adjust as needed)
SELECT pg_sleep(5);

-- Step 4: Drop the original table
DROP TABLE customers;

-- Step 5: Restore the table using AS OF SYSTEM TIME
CREATE TABLE customers_restored AS
SELECT * FROM customers AS OF SYSTEM TIME '2025-06-09 13:00:05.123456+00:00';
```

Reply:
ah nice, thanks for the example - added in https://github.com/cockroachdb/docs/pull/19674/commits/a37be228522603330486695dcd9587ad627ef693